### ISSUE TITLE
Clarification on release tags..

### ISSUE THREAD

**Issue by preston** [mitre/cql-translation-service#5]
Are the git tags used in this repository intended to mirror the underlying CQL language version? ..or is the software tag version unrelated to the CQL version?

**Comment by hadleynet** [comment]
The tags are intended to mirror the underlying CQL language version.

**Comment by preston** [comment]
Ok. Is a given version (say, 1.2) intended to also support older versions up through that release tag?

**Comment by hadleynet** [comment]
If the CQL source remains compatible then it should work - are you seeing something specific?

**Comment by preston** [comment]
No, it just needs some clarification in the documentation on which tags/branches are appropriate for a given CQL version. I'd suggest noting the approach in the README so people don't inadvertently deploy the wrong version(s). :)
